roles/virt-install/files/vm-4.ks

```diff
@@ -0,0 +1,62 @@
+#platform=x86, AMD64, or Intel EM64T
```

Comment:
these first 2 lines seem "unnecessary" and can be confusing. Maybe just remove? Also, I'd suggest adding a bit more "air" in-between lines to make this a bit easier to read - i.e.: add a newline before a comment section